distributed/worker_memory.py

```diff
@@ -28,6 +28,7 @@
 from collections.abc import Callable, MutableMapping
 from contextlib import suppress
 from functools import partial
+from time import monotonic
```

Comment:
On Windows, this has a granularity of 15ms. Could you use https://github.com/dask/distributed/pull/6181 instead?

Reply:
Maybe `distributed.metrics.time` would suffice?